### PR TITLE
add Kosovo as country

### LINKED ISSUE #4283: Add Kosovo as an independent country to profile
ZenDesk ticket at  https://odl.zendesk.com/agent/tickets/26911

blog post on ISO use of `XK`: https://geonames.wordpress.com/2010/03/08/xk-country-code-for-kosovo/

We'll need to fork https://github.com/olahol/iso-3166-2.js (again), but they do seem to accept PRs.

### PR DESCRIPTION
**Pull request by tasawernawaz**
Fixes https://github.com/mitodl/micromasters/issues/4283

### CODE REVIEW

File: data.csv [hunk 1]
```diff
@@ -2152,6 +2152,7 @@ Kiribati,KI-P,Phoenix Islands,Group of islands,KI
 "Korea, Republic of",KR-48,Gyeongsangnamdo,Province,KR
 "Korea, Republic of",KR-49,Cheju-do,Special self-governing province,KR
 "Korea, Republic of",KR-50,Sejong,Special self-governing city,KR
+kosovo,XK-XK,kosovo,country,XK
```
Comment (alicewriteswrongs) [inline data.csv]: should `kosovo` be capitalized here?

**Review by alicewriteswrongs — CHANGES_REQUESTED** [review]
one question!

File: data.csv (hunk 1 above)
Comment (tasawernawaz) [reply data.csv]: good catch, I will update it.

**Review by alicewriteswrongs — APPROVED** [review]
looks good to me!

**Pull request merged**